pretest.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+require('./test/init.js');
+
+var async = require('async');
+var fs = require('fs');
+var path = require('path');
+var _ = require('lodash');
+
+var db = global.getDataSource();
+var schemaName = global.config.schema;
+var sqlFile = path.resolve(__dirname, 'test', 'tables.dashdb.sql');
+
+if (!process.env.CI)
+  return console.log('Skipping pre-test ...');
+console.log('> Seeding the database ...');
+
+var contents = fs.readFileSync(sqlFile, {encoding: 'utf-8'});
+var lines = contents.trim().split(';');
```

Comment:
Have you considered instead just sending the entire file in a single `.execute()` call? The `ibm_db` module [supports it](https://github.com/ibmdb/node-ibm_db#-3-querysqlquery--bindingparameters-callback), but maybe the connector doesn't expose it?

Something like this, I imagine (untested, unverified):
```js
db.connector.query({
  sql: contents.replace(/"\?"/g, `"${schemaName}"`),
  noResults: true,
}, function(err, _noResults) {
  // done!?
});
```